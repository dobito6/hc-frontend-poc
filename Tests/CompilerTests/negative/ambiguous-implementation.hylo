//! no-std

trait P { fun f() }

given Void: P {
  fun f() {}
  fun f() {}
}
