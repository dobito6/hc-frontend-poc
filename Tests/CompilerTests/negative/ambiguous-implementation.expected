negative/ambiguous-implementation.hylo:5.1-6: error: ambiguous implementation of 'f'
given Void: P {
~~~~~
